samples/excel/20-data-types/data-types-error-values.yaml

```
@@ -53,16 +76,20 @@ script:
 template:
     content: |-
         <section class="ms-font-m">
-          <p>This sample shows how to set the value of cell A1 to the #BUSY! error data type.</p>
+          <p>This sample shows how to set the value of cell A1 to the #BUSY! error data type. The sample then checks to see if
+            cell A1 contains a #BUSY! error, and updates the cell value if it does.</p>
         </section>
         <section class="setup ms-font-m">
           <h3>Set up</h3>
           <button id="setup" class="ms-Button">
             <span class="ms-Button-label">Add worksheet</span>
           </button>
           <h3>Try it out</h3>
-          <button id="setBusyError" class="ms-Button">
-            <span class="ms-Button-label">Set A1 to busy error</span>
+          <button id="set-busy-error" class="ms-Button">
+            <span class="ms-Button-label">Set A1 to #BUSY! error</span>
+          </button>
+          <button id="change-busy-error" class="ms-Button">
+            <span class="ms-Button-label">Change output of cell with #BUSY! error</span>
```

The button text wraps in the standard Script Lab width, so consider making the text shorter.
```suggestion
            <span class="ms-Button-label">Change #BUSY! error cell value</span>
```